pages/_posts/2021-09-22-WebsiteRebuild.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: ""
```

Comment:
I don't know if a post needs a title :)

Reply:
Yes, it should. That was an oversight in my rush to get this done.